docs/guides/data-sync/version-control-guide.mdx

```diff
@@ -75,31 +75,103 @@ Once you are in the connect repository UI take the following steps:
 If you need to connect deployments to a private Github repo, provide an organization then click the Install Mage Pro app on Github button.
 </Note>
 
-## 2. Configure deployment stages:
+## 2 Sync to BitBucket 
+Mage Pro supports integration with BitBucket repositories for version control and collaboration. However, connecting to Bitbucket requires one additional configuration step before you can link your repository, you'll need to set up OAuth credentials as environment variables.
 
-Configure each stage of the deployment process before your code reaches production by completing the following steps:
+### Add / edit environment variables
```

Comment:
@colefreeman You can remove this entire section about adding the `BITBUCKET_OAUTH_KEY` and `BITBUCKET_OAUTH_SECRET` env vars. Users don't need to add any environment variables to use BitBucket deployments.